internal/generator/vector/filter/openshift/viaq/normalize.go

```diff
@@ -24,6 +24,8 @@ if !exists(.level) {
     .level = "info"
   } else if match!(.message, r'^D[0-9]+|level=debug|Value:debug|"level":"debug"') {
     .level = "debug"
+  } else if match!(.message, r'^D[0-9]+|level=trace|Value:trace|"level":"trace"') {
```

Comment:
I think the first regex should start with `T` for trace,
```suggestion
  } else if match!(.message, r'^T[0-9]+|level=trace|Value:trace|"level":"trace"') {
```

Reply:
nice catch